packages/header-component/src/components/dc-header/styles/base/_variables.scss

```diff
@@ -14,8 +14,15 @@ $color-primary: #fbfbfb;
 $color-secondary: #ff4630;
 
 $color-beige: #f9f8f5;
-$color-yellow: #ffed9c;
+$color-green-dark: #1e9981;
 $color-green: #24ba9d;
+$color-grey: #dadada;
+$color-white: #ffffff;
+$color-yellow: #ffed9c;
+
+$color-text: #434343;
+$color-text-light: #74736f;
+$color-text-disabled: #52575c;
```

Comment:
@duranmla - I wonder if we should set the variables as 'native css' 
and then bring them into the scss. 

Something like this?
https://codeburst.io/magic-of-css-sass-variables-b07ae8c1b775
https://codyhouse.co/blog/post/css-custom-properties-vs-sass-variables

Not a merge blocker, just wondering if we can get away with having some of our 'core styles' be   more universal across different code-bases. And this seems a possible step towards that?

Reply:
I am actually migrating out of the CSS variables.
_As I feel more natural to do SCSS variables if we are doing SCSS, but we can catch up on that idea later._